docs/concepts/serving/executor/add-endpoints.md

```diff
@@ -95,41 +162,42 @@ The `async` definition is optional.
 The endpoint signature looks like the following:
 
 ```python
-from typing import Dict, Union, List, Optional
-from jina import Executor, requests, DocumentArray
+from typing import Dict, Union, Optional
+from jina import Executor, requests
+from docarray import DocList
 
 
 class MyExecutor(Executor):
     @requests
     async def foo(
         self,
-        docs: DocumentArray,
+        docs: DocList[...],
```

Comment:
what is this `...` ?

Reply:
a place holder so that people understand is some generic

Reply:
I think it might be confusing with the python `...` syntax. Maybe we could create a a dummy base doc for this ?